src/api/dependency-discovery/test/dependency.test.js

```diff
@@ -14,9 +14,11 @@ const depsList = [
 ];
 
 jest.mock('../src/dependency-list', () => {
```

Comment:
The way you usually do this with Jest is to add a `__mocks__` directory, and write your overrides in a file there, see https://jestjs.io/docs/manual-mocks